File: 프로그래머스/파이썬/중복된 문자 제거.py
def solution(my_string):
    return ''.join(dict.fromkeys(my_string))

solution('people')

'''
아래 풀이도 참고!
'''

def solution(my_string):
    answer = ''
    for i in my_string:
        if i not in answer:
            answer+=i
    return answer

File: 코테 실습/JS/중복된 문자 제거.js
function solution(my_string) {
    var answer = '';
    
    for (let i = 0; i < my_string.length; i++) {
        const char = my_string[i];
        
        if (!answer.includes(char)) {
            answer += char;
        }
    }
    return answer;
}